keept.c

```diff
@@ -1071,7 +1074,9 @@ int main(int argc, const char * argv[])
 	die("Connecting to socket failed:");
     }
     G.redraw_mode = G.redraw_mode | 128; // bit 7 for immediate first connection
-    return attached(s);
+    int aret = attached(s);
+    if(remove_socket) unlink(sockname);  // ignore any error (socket file may has been moved)
```

Comment:
space after if -- if (remove_socket)

also, I think it should say '(socket file may have been moved)'